src/ui/public/styles/base.less

```diff
@@ -232,6 +232,11 @@ a {
     background-color: transparent;
     border-radius: 0;
   }
+
+  i.time-nav-btn {
```

Comment:
This can just be `.time-nav-btn` because using the element selector increases the selector's specificity unnecessarily.